### PR TITLE
DAOS-6194 test: Avoid datamover timeout

### PR DESCRIPTION
**Pull request by daltonbohning**
Test-tag: copy_procs

Quick-build: true
Skip-func-hw-test: true

For the copy_procs test, reduced the process list
from [3, 7, 16] to [3, 5, 8].
This reduces contention while still testing the same
functionality, so the test does not timeout.
The file size was adjusted accordingly.

Also made some minor style updates per request.

Signed-off-by: Dalton Bohning <daltonx.bohning@intel.com>

### CODE REVIEW

**Comment by daltonbohning** [comment]
@saurabhtandan A note about the timeout:
We previously thought we could reduce the 600s timeout, but even with 600s, the test timed out on Leap 15. With these changes, the latest run took ~150s. We can probably reduce the timeout now, but I would like to let this run on some master builds before making the decision, since the time varies between different runs (several processes on a single node).

**Review by shimizukko — APPROVED** [review]
LGTM

File: src/tests/ftest/datamover/copy_procs.py [hunk 1]
```diff
@@ -46,46 +44,47 @@ def setUp(self):
...
-        self.test_file = self.params.get(
-            "test_file", "/run/ior/*")
-        self.flags_write = self.params.get(
-            "flags_write", "/run/ior/copy_procs/*")
-        self.flags_read = self.params.get(
-            "flags_read", "/run/ior/copy_procs/*")
+        self.ior_flags = self.params.get(
+            "ior_flags", "/run/ior/*")
+        self.flags_write = self.ior_flags[0]
+        self.flags_read = self.ior_flags[1]
```
Comment (saurabhtandan) [inline src/tests/ftest/datamover/copy_procs.py:50]: Optional: We can avoid the use of these extra params by directly using self.ior_flags[0] and self.ior_flags[1]

**Review by saurabhtandan — APPROVED** [review]
LGTM

**Pull request merged**